src/action/common/create_users_and_groups.rs

```diff
@@ -37,7 +35,7 @@ impl CreateUsersAndGroups {
                     settings.nix_build_group_id,
                 )
             })
-            .collect();
+            .collect::<Result<_, _>>()?;
```

Comment:
That's a weird type signature... I don't often see `collect`ing into `Result<_, _>` :laughing:

Reply:
It's fun to can turn a `Vec<Result<_,_>>` into a `Result<Vec<_>,_>`